tests/test_run_examples.py

```diff
@@ -224,7 +224,10 @@ def test_run_directly(tmp_path, eval_example):
 x = 4
 
 def div(y):
-    return x / y
+    try:
+        return x / y
+    finally:
+        str(y)
```

Comment:
This is the line where the traceback would point without the change, as it's the last line the frame executed. tb_lineno points to where the exception happened, two lines up.